### DISCUSSION TITLE
Unknown object type: SparkJobDefinition

### DISCUSSION THREAD

**Discussion by cdawnjodata** [Azure-Player/azure.datafactory.tools#427] [Q&A] [answered]
I have a pipeline that calls a Spark Job Definition, and I'm getting the error: ADFT0029: Unknown object type: SparkJobDefinition
when attempting to Build (validate) or to Deploy, even if I exclude the pipelines. Does anyone know if this is supported or have a workaround? Thanks!

**Comment by NowinskiK** [comment, marked as answer]
Thanks for asking this question. I created a new issue (#428), which is now resolved.
